addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3098,6 +3099,20 @@ export class OdooEditor extends EventTarget {
                 fragment.appendChild(p);
             }
         }
+
+        // Split elements containing <br> into seperate elements for each line.
+        const brs = fragment.querySelectorAll('br');
+        for (let br of brs) {
+            const nearestBlock = closestBlock(br);
```

Comment:
change ` const nearestBlock = closestBlock(br);`  to ` const block = closestBlock(br);`